.github/workflows/golangci-lint.yml

```
@@ -35,6 +35,17 @@ jobs:
       - name: Install dependencies
         run: make check-tensorflow
       
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)